### PR TITLE
Translation of doc/quick-start.md

### PR DESCRIPTION
**Pull request by abhaypawar**
This is the translation for the quick start section for Gatsby Marathi.
 @spielers

### CODE REVIEW

**Review by spielers — DISMISSED** [review]
remove blank spaces

**Review by gamitics — DISMISSED** [review]
well, documented keep it up @abhaypawar

**Comment by abhaypawar** [comment]
Proofread and Edit done.

**Comment by spielers** [comment]
> remove blank spaces

k

**Review by abhaypawar — COMMENTED** [review]
Done.

**Comment by abhaypawar** [comment]
> remove blank spaces

File: docs/docs/quick-start.md [hunk 1]
```diff
@@ -2,67 +2,67 @@
...
-This quick start is intended for intermediate to advanced developers. For a gentler intro to Gatsby, [head to our tutorial](/tutorial/)!
+हे पृष्ठ इंटरमेडिएट आणि अडवान्सड डेव्हलपर्स यांना उद्देशून आहे. नवीन गॅटस्बी सभासदांनी परिचय पृष्ठ येथे पाहावे, [head to our tutorial](/tutorial/)!
 
-## Use the Gatsby CLI
+## गॅटस्बी CLI वापरा 
 
 <EggheadEmbed
   lessonLink="https://egghead.io/lessons/gatsby-quick-start-with-gatsby-create-develop-and-build-gatsby-sites-from-the-command-line"
-  lessonTitle="Quick Start with Gatsby: Create, Develop, and Build Gatsby Sites From the Command Line"
+  lessonTitle="गॅटस्बी सुरुवात: गॅटस्बी साईट वापरून करा -क्रीएट  डेव्हलप संचालन विविध कमांड लाईन वापरून."
```
Comment (visshaljagtap) [inline docs/docs/quick-start.md]: suggestion: कमांड लाइन वरून गॅट्सबी साइट्स तयार करा

**Comment by visshaljagtap** [comment]
@abhaypawar please resolve confict so that I can merge

**Review by visshaljagtap — APPROVED** [review]

**Comment by visshaljagtap** [comment]
> @abhaypawar please resolve confict so that I can merge

Did it

**Pull request merged**